src/util/specific.py

```diff
@@ -15,7 +15,7 @@
 def dbuuid(cr):
     cr.execute(
```

Comment:
Can then be written as
```suggestion
    return _dbuuids(cr)[-1]
```